### PR TITLE
Extend OpenFoam-Extend sanity check for decomp libaries

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

File: easybuild/easyblocks/o/openfoam.py [hunk 1]
```diff
@@ -323,10 +323,12 @@ def sanity_check_step(self):
                [os.path.join(toolsdir, "surface%s" % x) for x in ["Add", "Find", "Smooth"]] + \
                [os.path.join(toolsdir, x) for x in ["deformedGeom", "engineSwirl", "modifyMesh",
                                                     "refineMesh", "vorticity"]]
-        # check for the Pstream and scotchDecomp libraries, there must be a dummy one and an mpi one
+        # check for the Pstream and *Decomp libraries
         if 'extend' in self.name.lower():
-            libs = [os.path.join(libsdir, "libscotchDecomp.so")]
+            libs = [os.path.join(libsdir, "libscotchDecomp.so"), os.path.join(libsdir, "libmetisDecomp.so"),
+                    os.path.join(libsdir, "mpi", "libparmetisDecomp.so")]
```
Comment (boegel) [inline easybuild/easyblocks/o/openfoam.py]: hmm, why are we not checking this for both OpenFOAM and OpenFOAM-Extend?

Comment (wpoely86) [inline easybuild/easyblocks/o/openfoam.py]: Because `OpenFOAM` has issues of its own: hpcugent/easybuild-easyconfigs#2236

**Comment by wpoely86** [comment]
Tested in: https://gist.github.com/wpoely86/2eb37c3ab832073fea63

**Comment by boegel** [comment]
@wpoely86: merge is broken

**Comment by boegel** [comment]
@wpoely86: ping (needs sync with `develop`)

**Comment by wpoely86** [comment]
@boegel rereview please

**Comment by boegel** [comment]
@besserox: thoughts?

**Comment by besserox** [comment]
@wpoely86 It looks good to me.

One small remarks: 
Can you keep these comments around for both OpenFOAM and OpenFOAM-Extend?
```
# check for the Pstream and *Decomp libraries
# Pstream should have both a dummy and a mpi one
```

Thanks!

**Comment by wpoely86** [comment]
@besserox okay done

**Comment by boegel** [comment]
(close/open to trigger Travis)

**Comment by boegel** [comment]
@wpoely86 please review/merge https://github.com/wpoely86/easybuild-easyblocks/pull/26 ; without this, I couldn't verify this PR via `--module-only`

**Comment by boegel** [comment]
good to go, thanks @wpoely86!

**Pull request merged**